src/olympia/blocklist/views.py

```diff
@@ -132,6 +156,46 @@ def between(ver, min, max):
 
 
 @non_atomic_requests
+def blocklist_json(request):
+    key = 'blocklist:json'
+    cache.add('blocklist:keyversion', 1)
+    version = cache.get('blocklist:keyversion')
+    response = cache.get(key, version=version)
+    if response is None:
+        response = _blocklist_json(request)
+        cache.set(key, response, 60 * 60, version=version)
+    patch_cache_control(response, max_age=60 * 60)
+    return response
+
+
+def _blocklist_json(request):
+    items, _ = get_items()
+    plugins = get_plugins()
+    issuerCertBlocks = BlocklistIssuerCert.objects.all()
+    gfxs = BlocklistGfx.objects.all()
+    ca = None
+
+    try:
+        ca = BlocklistCA.objects.all()[0]
+        # base64encode does not allow str as argument
```

Comment:
We're quite far from python3 ready :) 

Yeah, I noticed afterwards that it was copied from the blocklist.xml view.